docs/spec/sources.md

```diff
@@ -345,3 +345,17 @@ this instantiation brings onto the eventing mesh. For a more detailed
 description, please refer to the
 [Event Registry](https://knative.dev/docs/eventing/event-registry/)
 documentation.
+
+## Source Event delivery
+
+Sources MUST output CloudEvents. The output MUST be via the HTTP binding specified in
```

Comment:
output -> produce. This is the language of the CloudEvents spec.

Reply:
the output might not be HTTP.

MUST be via the -> SHOULD be via the

Reply:
What could be the other outputs?

Reply:
My personal opinion about it is that we should just use in streaming the eventing sources with HTTP Cloudevents outputs, because outputting to different protocols could add a lot of burden to the shoulders of source developers, which could slow down the development of them